src/Groups/homomorphisms.jl

```diff
@@ -1298,7 +1297,7 @@ end
 Base.:^(x::GAPGroupElem{T},f::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup = apply_automorphism(f, x, true)
 #Base.:^(f::GAPGroupElem{AutomorphismGroup{T}},g::GAPGroupElem{AutomorphismGroup{T}}) where T <: GAPGroup = g^-1*f*g
 
-function (A::AutomorphismGroup{T})(f::GAPGroupHomomorphism{T,T}) where T <: GAPGroup
+function (A::AutomorphismGroup{S})(f::GAPGroupHomomorphism{T,T}) where S <: GAPGroup where T <: GAPGroup
```

Comment:
```suggestion
function (A::AutomorphismGroup{S})(f::GAPGroupHomomorphism{T,T}) where {S <: GAPGroup, T <: GAPGroup}
```
a double `where` is almost never necessary.
Alternatively, you could even do
```suggestion
function (A::AutomorphismGroup{<: GAPGroup})(f::GAPGroupHomomorphism{T,T}) where T <: GAPGroup
```

Reply:
o.k.
Concerning the multiple `where`, there are lots in many code files, this is perhaps something for a pull request of its own.